Framework/docs/tutorial/time-server.md

```diff
@@ -120,9 +129,18 @@ const debug = true; // shows when redraw is done
 mount(document.body, view, model, debug);
 ```
 
+This avoids inline `<script>` and works with the framework’s default Content Security Policy.
```

Comment:
Added a short CSP explanation to preempt confusion if someone tries to inline scripts and hits browser blocks.